build/commands/scripts/updatePatches.js

```diff
@@ -17,26 +17,19 @@ function loadChromiumPathFilter(filePath) {
 
   const prefixes = [];
   const suffixes = [];
-  const exactMatches = new Set();
 
-  // This should be revisited in the future once `path.matchesGlob` is stable
-  // and available in node to use, as this current implementation is a bit
-  // naive.
+  // This can be revisited once `path.matchesGlob` is available on stable node.
+  // This naive implementation is closer to how chromium handles .cfg
+  // exclusion files.
   for (const line of configLines) {
     if (line.startsWith('*')) {
       suffixes.push(line.slice(1));
-    } else if (line.endsWith('*')) {
```

Comment:
were these deleted lines just not needed anymore? If that's the case, maybe they should be removed separately from this PR to avoid confusion?

Reply:
I'll move this to another PR.